x/leverage/keeper/msg_server.go

```diff
@@ -195,10 +195,6 @@ func (s msgServer) Collateralize(
 		return nil, err
 	}
 
-	if err := s.keeper.checkCollateralLiquidity(ctx, types.ToTokenDenom(msg.Asset.Denom)); err != nil {
-		return nil, err
-	}
-
```

Comment:
We cannot remove this: `MinCollateralLiquidity` must prevent adding collateral when that would reduce the module's collateral liquidity below required value.